### PR TITLE
Add clearer error handling to confirmed preference save issue #170144565

### PR DESCRIPTION
**Pull request by akegan**
This PR also updates the way that we send preference updates to only include the total monthly rent if the preference is rent burdened

### CODE REVIEW

File: app/controllers/api/v1/applications_controller.rb [hunk 1]
```diff
@@ -16,8 +16,6 @@ def update
         response = rest_application_service.update(application_params.merge(id: params[:id]))
         if response
           render json: true
-        else
```
Comment (akegan) [inline app/controllers/api/v1/applications_controller.rb:19]: this is handled already by rails, and this way we provide a bit more info to the console.logs on the front end

**Comment by akegan** [comment]
still a bit of a work in progress, just need to add some rails tests

**Comment by akegan** [comment]
@LoganArnett this PR is ready for review

**Review by LoganArnett — APPROVED** [review]
Working as expected 👍

**Pull request merged**